spec/messaging_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "messaging"
+
+describe Message do 
+subject(:message) {described_class.new}
+
+it "should send a text message" do
+
+ thank_you_message = "Thank you! Your order was placed and will be delivered before #{one_hours_time}"
+ allow(message).to receive(:text_message) {thank_you_message}
+ expect(message.text_message).to eq thank_you_message
+ end
```

Comment:
Lint/BlockAlignment: `end` at 11, 1 is not aligned with `it "should send a text message" do` at 6, 0.